packages/core-commands/src/hooks.js

```diff
@@ -1,12 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { store as coreStore } from '@wordpress/core-data';
-import { useSelect } from '@wordpress/data';
-
-export function useIsBlockBasedTheme() {
```

Comment:
Since this hook was not exposed and felt redundant, I removed it and switched to an ad-hoc approach.